test/e2e_test.go

```diff
@@ -561,24 +561,24 @@ func consumeWithSideCarContainer(client clientset.Interface, podName string) err
 			{
 				Name:    podName,
 				Command: []string{"./consume-cpu/consume-cpu"},
-				Args:    []string{"--duration-sec=60", "--millicores=50"},
+				Args:    []string{"--duration-sec=600", "--millicores=50"},
```

Comment:
this shouldn't be needed since the test is only running for 60 seconds: https://github.com/kubernetes-sigs/metrics-server/blob/master/test/e2e_test.go#L173